core/module/Dog/dog_modules/dog_module/Shadowlamb/cmd/mount/mount.php

```diff
@@ -257,7 +257,7 @@ private static function on_pop(SR_Player $player, array $args)
 				return false;
 			}
 
-			if (false === $item->useAmount($player, $amt))
+			if (false === $item->useAmount($player, $amt, true, true))
```

Comment:
I don't think you'd want to have `modify = true` here. That should happen only on `#use`, if I understand this stuff correctly.

Reply:
True preserves the current behavior.

Reply:
Hmm, I don't think `modify()` does anything with mount, does it?

Also, `modify()` is called at the end of this function (line 299), after the `giveItem()` call, so having it called here has no real use, I'd say.

Reply:
You're right, @dloser, I'll consider calling `useAmount()` with `$modify=false`